src/components/Home/Header/index.tsx

```diff
@@ -114,94 +124,130 @@ const logosData: Array<{
     widthMd: 88,
     widthLg: 124,
     src: rabbitmqLogo,
-    alt: 'RabbitMQ logo'
+    alt: 'RabbitMQ logo',
+    link: '/doc/user-guide/serving/rabbitmq'
   },
   {
     widthSm: 66,
     widthMd: 88,
     widthLg: 124,
     src: pythonLogo,
-    alt: 'Python logo'
+    alt: 'Python logo',
+    link: '/doc/user-guide/building/pip'
```

Comment:
Now it's link to https://mlem.ai/doc/user-guide/building/pip. Maybe it should be a link to https://mlem.ai/doc/user-guide/models

Reply:
Existing link (pip) makes more sense to me than models tbh